charts/nexus-repository-manager/values.yaml

```diff
@@ -30,18 +32,16 @@ nexus:
       # nexus.ldap.env.java.naming.security.authentication: simple
   # nodeSelector:
   #   cloud.google.com/gke-nodepool: default-pool
-  resources: {}
+  resources:
```

Comment:
Is this change right?  Isn't it supposed to follow by definition (whereas before it was empty)?

Reply:
yeah, newer helm is OK with not having that place holder.  it continues to work fine with it gone, and it's one less thing to modify when you choose to uncomment something below.

Reply:
Ah nice, thanks for the lesson :)